package.json

```diff
@@ -7,8 +7,9 @@
   "private": true,
   "resolutions": {
     "moment": "^2.20.1",
-    "d3-scale/d3-array": "2.3.3",
-    "vfile-message": "^2.0.2"
+    "d3-scale>d3-array": "2.3.3",
+    "vfile-message": "^2.0.2",
+    "@types/node": "^14.6.2"
```

Comment:
https://stackoverflow.com/questions/64924337/i-am-getting-an-error-when-compiling-angular-6-project
typescript didnt understood the types
of the 12 or 13 version